packages/web/src/components/track/DownloadSection.tsx

```diff
@@ -192,9 +193,23 @@ export const DownloadSection = ({ trackId }: DownloadSectionProps) => {
     }
 
     return (
-      <Button variant='secondary' size='small' onClick={handleDownloadAll}>
-        {messages.downloadAll}
-      </Button>
+      <Tooltip
+        mount='body'
+        placement='left'
+        text={messages.followToDownload}
+        disabled={!shouldDisplayDownloadFollowGated}
+      >
+        <Flex onClick={(e) => e.stopPropagation()}>
```

Comment:
just curious why we need a flex?

Reply:
the pointer events are disabled on the disabled button, so the tooltip would never show.